pyEpiabm/pyEpiabm/tests/test_unit/test_intervention/test_social_distancing.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+from unittest.mock import patch
+
+import pyEpiabm as pe
+from pyEpiabm.property import InfectionStatus
+from pyEpiabm.intervention import SocialDistancing
+from pyEpiabm.tests.test_unit.parameter_config_tests import TestPyEpiabm
+
+
+class TestSocialDistancing(TestPyEpiabm):
+    """Test the 'SocialDistancing' class.
+    """
+    @classmethod
+    def setUpClass(cls) -> None:
+        """Intialise a population with one infector and one
+        infectee, both in the same place and household.
+        """
+        super(TestSocialDistancing, cls).setUpClass()  # Sets up parameters
+        cls.time = 1
+        cls.pop_factory = pe.routine.ToyPopulationFactory()
+        cls.pop_params = {"population_size": 1, "cell_number": 1,
+                          "microcell_number": 1, "household_number": 1}
+        cls._population = cls.pop_factory.make_pop(cls.pop_params)
+        cls.microcell = cls._population.cells[0].microcells[0]
+        cls.person = cls.microcell.persons[0]
+        cls.person.update_status(InfectionStatus(7))
+
+        params = pe.Parameters.instance().intervention_params[
+            'social_distancing']
+        cls.socialdistancing = SocialDistancing(population=cls._population,
+                                                **params)
+
+    def test__init__(self):
+        self.assertEqual(self.socialdistancing.start_time, 6)
```

Comment:
Are these valules default in the function, or from the params file? Would be good to document what you're actually testing here!

Reply:
They are from params file, I added a comment line above to explain.

Reply:
Fixed!